src/test/java/com/google/devtools/build/lib/bazel/bzlmod/BazelLockFileModuleTest.java

```diff
@@ -0,0 +1,99 @@
+package com.google.devtools.build.lib.bazel.bzlmod;
```

Comment:
please add the usual copyright comment block here

Reply:
Done